test/composables/useTheme.test.ts

```
@@ -456,3 +456,166 @@ describe('server configuration', () => {
+  it('returns the default code samples generator', () => {
+    const result = themeConfig.getCodeSamplesGenerator()
+    expect(result).toBeTypeOf('function')
+  })
+
+  it('returns the default code samples default headers', () => {
+    const result = themeConfig.getCodeSamplesDefaultHeaders()
+    expect(result).toEqual({})
+  })
+
+  it('sets and gets the code samples config', () => {
+    const customGenerator = (lang: string, request: any) => Promise.resolve(`Custom code for ${lang}`)
```

ESLint

[error] 'request' is defined but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)